### DISCUSSION TITLE
文档总纲

### DISCUSSION THREAD

**Discussion by zuoez02** [siyuan-community/siyuan-developer-docs#1] [Ideas]
有什么文档总纲的想法？这样方便分领任务。

**Comment by Zuoqiu-Yingyi** [comment]
当然是参考一下这种了\~
- [Obsidian 插件开发文档 | Obsidian 插件开发文档](https://luhaifeng666.github.io/obsidian-plugin-docs-zh/)
- [Obsidian Plugin Developer Docs | Obsidian Plugin Developer Docs](https://marcus.se.net/obsidian-plugin-docs/)
